etc/modules.d/exec-wait.yaml

```diff
@@ -5,4 +5,4 @@ help: Enable exec-wait binary for shared-run mode.
 env: ENABLE_EXEC_WAIT
 copy: []
 bind:
-  - ../../bin/exec-wait:/usr/bin/exec-wait
+  - /usr/bin/exec-wait:/usr/bin/exec-wait
```

Comment:
I would prefer to leave this repo version as a relative path (or it will break dev installs in conda venvs), and version our site-specific paths in a separate repo.

This module is also a bit different that `gpu` and `mpich` since the binary is getting installed with the python lib, and doesn't reference external libraries.  This module should also always be installed for all sites, since `shared-run` subcommand depends on it.

Reply:
Ok, I'll put the relative paths back for now. We'll have to figure out how to reconcile our system specific configuration with this configuration, but we can do that later.

Reply:
Sorry, this is my mistake.  This path should be 
`../../../bin/exec-wait:/usr/bin/exec-wait`